src/blue-theme.scss

```diff
@@ -43,3 +43,9 @@ $primary: $democracy-blue;
 [class*="toast"] {
     margin-bottom: 0 !important;
 }
+
+// wraps text that has whitespace (handy for column header and labels)
+[class*="wrap"] {
+    white-space: normal;
+    height: auto;
+}
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5707004)

Reply:
@oysor can you address this? You should be getting the emails too about these issues.

Reply:
When you appease Codacy I think we can approve this.